packages/apps/job-launcher/server/src/modules/job/job.entity.ts

```diff
@@ -28,6 +28,9 @@ export class JobEntity extends BaseEntity implements IJob {
   @Column({ type: 'varchar' })
   public manifestHash: string;
 
+  @Column({ type: 'varchar', nullable: true })
+  public latestTransactionHash: string;
```

Comment:
I think checking the cancelled status of the subgraph would be a good idea instead of saving the hash of the cancelled transaction in the database and checking it later. We can avoid having a nullable field in the database that is going to be mostly null.

cc: @portuu3, @leric7